packages/preset-dumi/src/utils/getHostPkgAlias.ts

```diff
@@ -24,11 +25,14 @@ export default (paths: IApi['paths']) => {
     const { version: lernaVersion } = require('lerna/package.json');
 
     if (lernaVersion.startsWith('3')) {
-      JSON.parse(
-        execSync(`${path.join(paths.cwd, 'node_modules/.bin/lerna')} ls --json --all`, {
-          stdio: 'pipe',
-        }).toString(),
-      ).forEach(pkg => {
+      let res = execSync(`${path.join(paths.cwd, 'node_modules/.bin/lerna')} ls --json --all`, {
+        stdio: 'pipe',
+      }).toString();
```

Comment:
扔掉中括号后面的内容吧，加上注释描述为什么需要这么处理，这样覆盖率不会降低
```suggestion
      }).toString().replace(/([\r\n]\])[^]*$/, '$1');
```

Reply:
是的，输出是这样的，好的，我这么改一下